assets/src/edit-story/elements/video/index.js

```diff
@@ -26,6 +26,9 @@ export const defaultAttributes = {
 	controls: false,
 	loop: false,
 	autoPlay: true,
+	featuredMedia: 0,
```

Comment:
IMHO, `null` or `undefined` would go better in place of surrogate "n/a"  values. Any reason for them?

Also, why not just call them here "poster". Is there any HTML-based format that prefers "featuredMedia" naming? If it's just WP, maybe better to stop propagation of this naming at API or Media Library?

Reply:
I have changed the naming to `poster` and `posterId`. 

The reason it is 0, is because the type is number `posterId: PropTypes.number`

Reply:
As long as it's not `PropTypes.number.isRequired`, `null` works

Reply:
Could also use `PropTypes.oneOfType( [ ... ] )` perhaps.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/135/commits/7a1c40a11e2ea3824f36cb17ad998ce54bcfb343

Reply:
FWIW, I usually use `string` for any ID. IMHO, the numeric constraint is coincidental to what an ID could possibly be. Could be relevant once we try to separate WP APIs further to allow standalone functioning. But we could review/solve it then.